### PR TITLE
feat: add slider component

### PR DESCRIPTION
**Pull request by Maianki**
## Added slider component
Screenshot (617)

## Live link
https://happy-brattain-053776.netlify.app/

### CODE REVIEW

**Review by sainath7878 — APPROVED** [review]
Everything looks good to merge.

**Comment by Maianki** [comment]
> Everything looks good to merge.

Thanks for reviewing.

**Review by yash9263 — APPROVED** [review]

**Comment by yash9263** [comment]
Everything looks good to me

**Comment by Maianki** [comment]
> Everything looks good to me

Thanks for reviewing.

File: components/components-styles/slider.css [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+    background:var(--GRAY-COLOR);
+    outline: none;
+    border-radius: 2px;
+    opacity: 0.8;
+  }
+
+.slider-rating::-webkit-slider-thumb,
+.slider-disabled::-webkit-slider-thumb{
+    -webkit-appearance: none;
+    appearance: none;
```
Comment (theVivek-kumar) [inline components/components-styles/slider.css:21]: THIS PART IA OKAY

File: docs.html [hunk 2]
```diff
@@ -2215,6 +2216,96 @@ <h2 class="component-subheading">Three Items in a grid</h2>
...
+        <p class="description">This is a normal slider which works between a range of 0 to 100.  The slider changes with a step of 1, You need to add <span class="text-highlight">slider</span> and <span class="text-highlight">slider-range</span> classes to use it.
+        </p>
+
+        <section class="example-container align-items-center">
+
+            <label class="form-label" for="slider">
+            <input id="slider" type="range" min="1" max="100" value="50" />
+            
+            <div class="slider-range"> 
+                <span class="text-sm">1</span>
```
Comment (theVivek-kumar) [inline docs.html:2235]: INSERT LIVE LINK AND SCREENSHOTS IN  THIS PR

File: docs.html [hunk 3]
```diff
@@ -2215,6 +2216,96 @@ <h2 class="component-subheading">Three Items in a grid</h2>
             </pre>
         </section>
 
+        <hr>
+
+        <h1 id="Slider" class="component-heading">Slider</h1>
```
Comment (theVivek-kumar) [inline docs.html:2221]: TRY TO USE GOOD CLASS NAME

**Review by theVivek-kumar — COMMENTED** [review]
LOOKS GOOD TO ME

**Pull request merged**